File: pe-03/practice.js
const lessonMinutesText = "45";
const practiceMinutesText = "35";

const lessonMinutes = Number(lessonMinutesText);
const practiceMinutes = Number(practiceMinutesText);

if (Number.isNaN(lessonMinutes) || Number.isNaN(practiceMinutes)) {
    console.log("خطأ: قيمة الدقائق غير صالحة");
} else {
    const totalMinutes = lessonMinutes + practiceMinutes;
    const hours = Math.floor(totalMinutes / 60);
    const remainingMinutes = totalMinutes % 60;

    console.log(`${hours} ساعة و${remainingMinutes} دقيقة`);
    
}

File: pe-03/practice.py
lesson_minutes_text = "45"
practice_minutes_text = "35"

try:
    lesson_minutes = int(lesson_minutes_text)
    practice_minutes = int(practice_minutes_text)

    total_minutes = lesson_minutes + practice_minutes
    hours = total_minutes // 60
    remaining_minutes = total_minutes % 60

    print(f"{hours} ساعة و{remaining_minutes} دقيقة")

except ValueError:
    print("خطأ: قيمة الدقائق غير صالحة")
    